### DISCUSSION TITLE
Sync actions in Profiles

### DISCUSSION THREAD

**Discussion by NCLnclNCL** [kaliiiiiiiiii/Selenium-Profiles#83] [Q&A] [answered]
i don't know if selenium can do it, but i want it to be like android emulator, when i control 1 main emulator, other emulators will do the same, can you make Profile in selenium like that??

**Comment by ttlns** [comment, marked as answer]
no, not directly, as different browser windows might have different loading times.
I'd recommend you to use multithreading

@NCLnclNCL please mark this as the answer, if it resolves your question

**Comment by ttlns** [comment]
Closing as it seems to be resolved
